kpoof

```diff
@@ -88,15 +88,18 @@ po_select() {
   po_list | sed -n ${POD:-1}p
 }
 
-po_port_list() {
+port_list() {
   kubectl get po $(ns_param) $(po_select)\
   -o=jsonpath='{..containerPort}' \
   | tr " " "\n" \
   | sort -u
 }
 
-po_port_x() {
-  for i in $(po_port_list); do
+# Because \*nix denies binding to ports below 1001, we assign a port of N+50000,
```

Comment:
This may be copy pasta; but in a readme, the * has to be identified as a special character, but in bash this row is already commented out.  Can we remove the `\`?